### PR TITLE
Added responsive navbar

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
Signed-off-by: Prakhar Agarwal <prakhar@everything.net.in>

**Description**
Adds responsive navbar for mobile view

**Notes for Reviewers**

**Before:**

https://user-images.githubusercontent.com/17739006/145674339-10de5c04-7fa8-4353-9cb8-420f12fda0ef.mp4

**After:**

https://user-images.githubusercontent.com/17739006/145674282-15c496c9-2f9f-46c5-8a03-3d10c7acb5b1.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Header/Header.styles.js [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
+  toolbarOnDrawerOpen: {
+    minHeight: 58,
+    paddingLeft: 20,
+    paddingRight: 20,
+    [theme.breakpoints.between(620, undefined)]: { minHeight: 68, paddingLeft: 20, paddingRight: 20 },
+  },
+  itemActiveItem: { color: "#00B39F" },
+  headerIcons: { fontSize: "1.5rem", height: "1.5rem", width: "1.5rem" },
+});
+export const useStyles = makeStyles(getStyles);
```
Comment (bariqhibat) [inline ui/components/Header/Header.styles.js:36]: Does this get autoformatted?

**Review by bariqhibat — COMMENTED** [review]
This is such a great fix, thanks @Prakhar-Agarwal-byte! Does the rest of the code gets autoformatted by the precommit hooks?

File: ui/components/Header/Header.styles.js (hunk 1 above)
Comment (Prakhar-Agarwal-byte) [reply ui/components/Header/Header.styles.js:36]: Yes

**Review by sayantan1413 — APPROVED** [review]
LGTM

**Review by Abhishek-kumar09 — APPROVED** [review]
LGTM, Thanks @Prakhar-Agarwal-byte

**Comment by sudo-NithishKarthik** [comment]
@Prakhar-Agarwal-byte , i've recently made [this](https://github.com/meshery/meshery/blob/meshery-ui-restructuring/ui/package.json#L20) change which would make sure we don't have this much style change being added to the commit. Can you rebase your PR and then update it so that we don't get these unwanted formatting changes ?

**Comment by Prakhar-Agarwal-byte** [comment]
@sudo-NithishKarthik Done

**Review by sudo-NithishKarthik — APPROVED** [review]
Thanks @Prakhar-Agarwal-byte . Looking Good 🎉

**Pull request merged**